skills/ad4m-executor/SKILL.md

```
@@ -0,0 +1,244 @@
+**Always use the mainnet seed for development:**
+```bash
+ad4m-executor init --data-path /tmp/ad4m-data \
+  --network-bootstrap-seed ./cli/mainnet_seed.json
+```
+
+Without `--network-bootstrap-seed`, the executor looks for `mainnet_seed.seed` in the data path — a file that doesn't exist after init. This fails silently: the executor starts, agent generates, but all languages fail to resolve.
+
+### Diagnosing language failures
+
+If logs show:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)